jdaviz/configs/default/plugins/viewers.py

```diff
@@ -191,6 +191,23 @@ def _on_subset_create(self, msg):
         if msg.subset.label not in self._expected_subset_layers and msg.subset.label:
             self._expected_subset_layers.append(msg.subset.label)
 
+    def _on_subset_delete(self, msg):
+        if self.__class__.__name__ == 'MosvizTableViewer':
```

Comment:
Why not use `isinstance`?

Reply:
Because I was lazy and copied this from the existing `_on_subset_create`....I'll go ahead and update both.

Reply:
there might be a race condition on the imports... but if so, let's add a comment

Reply:
CI is failing with the `isinstance` change, but might just need `jdaviz.configs.mosviz.plugins.viewers` (plugins is missing).

Reply:
Whoops, fixed.